content/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -339,71 +339,72 @@ Once your CNAME is set up with your DNS provider, instead of providing an existi
 
 ---
 ## Step by Step Guide with Fastly
+_Updated June 2023. Images and descriptions follow the current Fastly web UI._
 
-Sign up for Fastly or log in to an existing account.
+Sign up for [Fastly](https://www.fastly.com/) or log in to an existing account.
 
-1. Select the **Configure** tab on the Dashboard, then "Create Service". Give your service a name, and add your tracking domain under "Domains".
+1. Select the **Deliver** tab on the Dashboard, then click the **Create a delivery service** button. Give your service a name in the _Options_ menu and clicking **Edit service name**.
```

Comment:
nitpicky - Give your service a name in the Options menu `by` clicking Edit service name.

Reply:
Done